reana_job_controller/rest.py

```diff
@@ -194,6 +194,10 @@ def create_job():  # noqa
     compute_backend = job_request.get(
         'compute_backend',
         current_app.config['DEFAULT_COMPUTE_BACKEND'])
+    if not current_app.config['MULTIPLE_COMPUTE_BACKENDS'] and \
+       current_app.config['DEFAULT_COMPUTE_BACKEND'] != compute_backend:
+        msg = 'Job submission to {} is not allowed'.format(compute_backend)
+        return jsonify({'job': msg}), 500
```

Comment:
Thanks, works well, although the "failed" reason is not visible to the end user:

```
$ reana-client status -w htc
NAME   RUN_NUMBER   CREATED               STATUS   PROGRESS
htc    6            2019-07-19T15:39:50   failed   0/1

$ reana-client logs -w htc
<nothing>
```

We can improve it later.